staging/src/k8s.io/cri-api/pkg/apis/testing/fake_runtime_service.go

```diff
@@ -327,7 +330,8 @@ func (r *FakeRuntimeService) CreateContainer(podSandboxID string, config *runtim
 			Labels:      config.Labels,
 			Annotations: config.Annotations,
 		},
-		SandboxID: podSandboxID,
+		SandboxID:      podSandboxID,
+		LinuxResources: config.GetLinux().GetResources(),
```

Comment:
What will be the contents of `config.GetLinux().GetResources()`?

Reply:
Basically, the cgroups configuration for the container
https://github.com/kubernetes/cri-api/blob/018cda79f51e06660614a18cb043661f111099e4/pkg/apis/runtime/v1alpha2/api.pb.go#L1338